homeassistant/components/onewire/onewire_entities.py

```diff
@@ -39,10 +39,10 @@ def __init__(
         self._device_class = SENSOR_TYPES[entity_type][1]
         self._unit_of_measurement = SENSOR_TYPES[entity_type][0]
         self._device_info = device_info
-        self._state = None
-        self._value_raw = None
+        self._state: int | bool | float | None = None
```

Comment:
From `homeassistant.helpers.typing`.
```suggestion
        self._state: StateType = None
```